docs/team/kimshitong.md

```diff
@@ -5,35 +5,82 @@ title: Shi Tong's Project Portfolio Page
 
 ### Project: MediConnect
 
-AddressBook - MediConnect is a desktop address book application used for managing hospital staff and patients. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+AddressBook - MediConnect is a desktop address book application used for managing hospital staff and patients. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 21 kLoC.
```

Comment:
change AddressBook to MediConnect. change address book application to clinic management application

Reply:
Ok, thanks for reminding!

Reply:
May I clarify this, should we say managing **clinic** staff and patients?

Reply:
yes should change that also

Reply:
great, thanks!